### ISSUE TITLE
Which checkpoints were used for final evaluation?

### ISSUE THREAD

**Issue by Yura52** [xu-ji/IIC#91]
Hi! All training scripts produce two checkpoints: "latest.pytorch" and "best.pytorch". Could you clarify, please, which of them is used to report figures in the paper?

**Comment by xu-ji** [comment]
I believe it was "best". (Since accuracy increases during training and approximately plateaus, there should not be a large difference.)
